src/github_runner_image_builder/cli.py

```diff
@@ -247,6 +247,12 @@ def _parse_url(
     help="Run an external bash setup script fetched from the URL on the runners during cloud-init."
     "Installation is run as root within the cloud-init script after the bare image default setup.",
 )
+@click.option(
+    "--script-secrets",
+    default="",
+    help="The space separated external secrets to load before running external script_url. e.g. "
+    "SECRET_ONE=HELLO SECRET_TWO=WORLD",
+)
```

Comment:
Passing secrets through CLI is not really a good practice as it might disclose them through the bash history, but also in the process cmd.
It's better to read the secrets from environment variables (still not ideal).

I'd suggest either to:
- Define a convention and for example look for all environment variables using a prefix and consider them as secrets (e.g all envvars starting with "SECRET_")
- Have a CLI argument that define which environment variable name to read, and get the value from the envvar
- Define the secrets in a file (with properly set permissions) and use the CLI to point to the file where secrets will be read